.gemspec

```diff
@@ -38,6 +38,7 @@ Gem::Specification.new do |s|
   # i18n support (gettext-setup and dependencies)
   s.add_runtime_dependency(%q<fast_gettext>, "~> 1.1.2")
   s.add_runtime_dependency(%q<locale>, "~> 2.1")
+  s.add_runtime_dependency(%q<multi_json>, "~> 1.10")
```

Comment:
Curious why 1.10 as opposed to a newer version?

Reply:
I took this out of Patrick's benchmarks PR, where he was also bumping this gem. This will still install the latest thing in 1.x series (I got 1.13.1 on my machine). I don't have a particular reason for not specifying 1.13 though. @pcarlisle did you?

Reply:
no